explore-analyze/report-and-share/automating-report-generation.md

```diff
@@ -159,3 +165,65 @@ If you experience issues with the deprecated report URLs after you upgrade {{kib
+* Sharing reports outside of {{kib}} requires a default preconfigured email connector.
+
+   * **{{ech}} or {{serverless-short}} users**: You do not need to set up a default preconfigured email connector. Kibana provides you with a built-in preconfigured email connector that uses the SMTP protocol to send emails. To view it, go to the **Connectors** page and find the Elastic-Cloud-SMTP connector.
+   * **Self-managed users**: You must set up a default preconfigured email connector to send notifications outside of {{kib}}. To do this:
+     
+     1. Open your `kibana.yml` file and add a new section for the `xpack.actions.preconfigured` setting. This setting specifies configuration details that are specific to the type of preconfigured connector you're defining. 
+     2. Under the `xpack.actions.preconfigured setting`, define the email connector. For example: 
+
+        ````
+        xpack.actions.preconfigured:
+          my-email:
+          name: preconfigured-email-connector-type
+          actionTypeId: .email
+          config:
+            service: other
+            from: testsender@test.com
+            host: validhostname
+            port: 8080
+            secure: false
+            hasAuth: true
+          secrets:
+            user: testuser
+            password: passwordkeystorevalue
+         `````
+
+         :::{tip} 
+         Refer to [Email connectors](kibana://reference/connectors-kibana/pre-configured-connectors.md#preconfigured-email-configuration) to learn about requirements for different email services and providers.
+         :::
+
+### Create a schedule [create-scheduled-report]
+
+1. Open the saved Discover session, dashboard, or visualization you want to share. 
+2. Click the **Export** icon, then **Schedule export**.
+3. Enter the requested details, and (optional) enable **Print format** to generate the report in a printer-friendly format.
+4. Set up a schedule for generating the report.
+
+    * **Date**: Choose when to start generating reports.
+    * **Timezone**: Specify a timezone for the schedule.
+    * **Repeat**: Choose how often you want to generate reports.  
+
+5. (Optional) To share generated reports outside of Kibana, enable **Send by email** and enter a list of email addresses. Recipients will receive emails with the generated reports attached and on the schedule that you specified.
+6. Click **Schedule exports** to save the schedule. 
+
+A message appears, indicating that the schedule is available on the **Reporting** page. From the **Reporting** page, click on the **Schedules** tab to view details for the newly-created schedule. 
+
+::::{important} 
+Note that you cannot edit or delete a schedule after you create it. To stop the schedule from running, you must disable it. Disabling a schedule permanently stops it from running. To restart it, you must create a new schedule. 
```

Comment:
Should we add a section on limitations? For example, we only allow attachments up to a certain size. Also, because we run reporting tasks one at a time, if they schedule too many reports for the same time, they may receive their reports on a delay. In general, if their cluster is busy with a lot of tasks, their reports may be delayed.

Reply:
Added in [006df70](https://github.com/elastic/docs-content/pull/2118/commits/006df70eab1ba3e54d62a1d1a40b34b9d4a2c7f9).